karate-core/pom.xml

```diff
@@ -113,7 +114,12 @@
             <artifactId>junit-jupiter</artifactId>
             <version>${junit5.version}</version>
             <scope>test</scope>
-        </dependency>		
+        </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
```

Comment:
what's this for - can we avoid this

Reply:
I found that it was necessary for `import javax.xml.bind.annotation.XmlRootElement` in `karate-core/src/test/java/com/intuit/karate/core/Cat.java`

Reply:
I see, thanks. yes I haven't got around to looking at the examples yet

Reply:
I never see an error here. at the very least please make this `test` scope, or remove the annotation

Reply:
nevermind, I removed the annotation: https://github.com/intuit/karate/commit/26aea084ed2acdf3b26422951718a7048a2be7d7